front/src/applications/editor/tools/routeEdition/components.tsx

```diff
@@ -41,7 +41,7 @@ export const RouteEditionLayers: FC = () => {
     <>
       {/* Editor data layer */}
       <GeoJSONs
-        selection={!isEmpty(selectionList) ? selectionList : ['placeholder']}
+        selection={!isEmpty(selectionList) ? selectionList : undefined}
```

Comment:
I think this is better because otherwise, if you have selected 1 item, then everything will be greyed again
```suggestion
        selection={selectionList.length > 1 ? selectionList : undefined}
```